CHANGELOG.md

```diff
@@ -22,6 +22,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Singletons authority and state resumes correct when reconnecting servers to snapshot.
 - Fixed a crash when retrying reliable RPCs with UObject arguments that got destroyed before the RPC was retried.
 - Fixed path naming issues in setup.sh
+- Made Reliable Client/Server RPCs use Events instead of Commands.
```

Comment:
This is the implementation - however if this is a user facing changelog then the big difference is guaranteeing ordering and reliability for single worker reliable rpcs.

Reply:
Will change it to "Ordering and reliability for single worker reliable RPCS are now guaranteed."

Reply:
Should we also mentioned that "Ordering and reliability for single worker unreliable RPCS are now guaranteed as well"? As it's almost the same execution flow.

Reply:
No. That is purely an implementation detail and it's not in anyone's interest to keep it long term.
It's not something that should be relied on long term.

Reply:
Changed as discussed.